test/testdata/lsp/highlight_untyped.rb

```diff
@@ -79,30 +80,11 @@ class Derived < Base
   sig { override.returns(String) }
   def foo
     super
-#   ^^^^^ untyped: This code is untyped
+#   ^^^^^ untyped: Value returned from method is `T.untyped`
   end
 end
 
 sig { params(x: Integer, y: String).returns(Integer) }
 def binary_method(x, y)
   4
 end
-
-untyped_array_args = [T.let(1, T.untyped), T.let(1, T.untyped)]
-# FIXME this should report an untyped error
-binary_method(*untyped_array_args)
-
-untyped_args = T.unsafe([])
-#              ^^^^^^^^^^^^ untyped: This code is untyped
-
-# FIXME the first assertion is also weird and needs
-# to be fixed.
- binary_method(*untyped_args)
-#     untyped: This code is untyped
-#^^^^^^^^^^^^^^^^^^^^^^^^^^^^ untyped: This code is untyped
-
-begin
-# FIXME this should not be an untyped error
-rescue StandardError
-  #    ^^^^^^^^^^^^^ untyped: This code is untyped
-end
```

Comment:
Would it be a regression to delete those tests? I didn't manage to find other tests for call with splat in this PR

Reply:
These tests are redundant with the new tests in `test/testdata/infer/strong_splat.rb` and `test/testdata/infer/strong_rescue.rb`